style/styles.css

```diff
@@ -453,6 +449,129 @@ header {
   margin-top: 6px;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 50px;
+  left: 0;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background: #c1c7d0;
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
- [ ] [OPTIONAL] Good job trying to implement the changes requested by the previous reviewer. But I think you still need to make the smaller viewport responsive. The picture for a project has static width. It is not changing as the viewport size changes. Please make sure it is truly responsive by using `vw` or `%` for setting the width of elements. 
- [ ] [OPTIONAL] There is a gap between the edge of the viewport and the modal in the Figma design. Please give it a padding or margin to make it look like the design. 

<p align="center">&nbsp;
 
| Figma Design |   Your project |
| ---------- | ----------- |
 | ![Template 1-Mobile-Detail](https://user-images.githubusercontent.com/12524453/180617216-70a68ed9-a17d-4255-bebf-8ca9d7f971f6.png) | ![Screen Shot 2022-07-23 at 20 48 15](https://user-images.githubusercontent.com/12524453/180617222-1f96beab-8488-4524-856e-4c1c9b31dad7.png) |
</p>